source/leuven/credits.rst

```diff
@@ -4,7 +4,7 @@ Credits to use KU Leuven infrastructure
 =======================================
 
 KU Leuven uses a credit system to do accounting on the HPC systems it hosts.
-This is the case for its Tier-2 clusters, as well as the Tier-1 infrastructure.
+This is the case for its Tier-2 clusters.
 
 Information on how to use the credit system can be found in the :ref:`documentation
 on credit system basics <credit system basics>`.
```

Comment:
Thanks for bringing this under my radar. I would remove `credit_system_basics` because the `slurm-accounting` page, as you correctly mentioned, fully covers the same topic.